src/services/index.ts

```diff
@@ -7,7 +7,7 @@ import {
   getPriceEstimationFactory,
 } from './factories'
 import { logDebug } from 'utils'
-import { TokenDetails } from '@gnosis.pm/dex-js'
+import { TokenDetails } from 'types'
```

Comment:
Are they different?

Reply:
```ts
//dex-js
export interface TokenDetails {
    id: number;
    name?: string;
    symbol?: string;
    decimals?: number;
    address: string;
    addressMainnet?: string;
    image?: string;
}

//dex-react
export interface MinimalTokenDetails {
  address: string
  symbol?: string
  name?: string
  decimals: number
}

export interface TokenDetails extends MinimalTokenDetails {
  id: number
  addressMainnet?: string
  image?: string
}
```

`decimals` is required in dex-react
which probably doesn't matter much, but TS was complaining